Bottom: 5165d58bd1af334b97fd5a58916f37c12d06a4c0
Top:    cc2084ca8811d0f494a3a2eef67d06e16b1d2099
Author: Jon Smirl <jonsmirl@gmail.com>
Date:   2013-07-20 10:14:51 -0400

Refresh of asiarf

---

diff --git a/target/linux/ramips/dts/AWM002-4M.dts b/target/linux/ramips/dts/AWM002-4M.dts
deleted file mode 100644
index adadfa1..0000000
--- a/target/linux/ramips/dts/AWM002-4M.dts
+++ /dev/null
@@ -1,108 +0,0 @@
-/dts-v1/;
-
-/include/ "rt5350.dtsi"
-
-/ {
-	compatible = "AWM002", "ralink,rt5350-soc";
-	model = "AsiaRF AWM002";
-
-	palmbus@10000000 {
-		sysc@0 {
-			ralink,pinmux = "spi", "uartlite", "jtag", "mdio", "sdram", "rgmii";
-			ralink,gpiomux = "i2c";
-			ralink,uartmux = "gpio";
-			ralink,wdtmux = <1>;
-		};
-
-		gpio0: gpio@600 {
-			status = "okay";
-		};
-
-		spi@b00 {
-			status = "okay";
-			m25p80@0 {
-				#address-cells = <1>;
-				#size-cells = <1>;
-				compatible = "mx25l6405d";
-				reg = <0 0>;
-				linux,modalias = "m25p80", "mx25l6405d";
-				spi-max-frequency = <10000000>;
-
-		partition@0 {
-			label = "u-boot";
-			reg = <0x0 0x30000>;
-			read-only;
-		};
-
-		partition@30000 {
-			label = "u-boot-env";
-			reg = <0x30000 0x10000>;
-			read-only;
-		};
-
-		factory: partition@40000 {
-			label = "factory";
-			reg = <0x40000 0x10000>;
-			read-only;
-		};
-
-		partition@50000 {
-			label = "firmware";
-			reg = <0x50000 0x1fb0000>;
-		};
-			};
-		};
-	};
-
-	ethernet@10100000 {
-		status = "okay";
-	};
-
-	esw@10110000 {
-		status = "okay";
-		ralink,portmap = <0x3f>;
-	};
-
-	wmac@10180000 {
-		status = "okay";
-	};
-
-	ehci@101c0000 {
-		status = "okay";
-	};
-
-	ohci@101c1000 {
-		status = "okay";
-	};
-
-	i2c-gpio {
-		compatible = "i2c-gpio";
-
-		gpios = <&gpio0 1 0 &gpio0 2 0>;
-		i2c-gpio,delay-us = <10>;
-
-		pcf0: iexp@38 {
-			#gpio-cells = <2>;
-			compatible = "inxp,pcf8574a";
-			reg = <0x38>;
-			gpio-controller;
-		};
-
-		hwmon@4b {
-			compatible = "national,lm92";
-			reg = <0x4b>;
-		};
-	};
-
-	gpio-leds {
-		compatible = "gpio-leds";
-		ld1 {
-			label = "asiarf:green:ld1";
-			gpios = <&pcf0 0 1>;
-		};
-		ld2 {
-			label = "asiarf:green:ld2";
-			gpios = <&pcf0 1 1>;
-		};
-	};
-};
diff --git a/target/linux/ramips/dts/AWM002-4M.dtsi b/target/linux/ramips/dts/AWM002-4M.dtsi
new file mode 100644
index 0000000..a506b69
--- /dev/null
+++ b/target/linux/ramips/dts/AWM002-4M.dtsi
@@ -0,0 +1,18 @@
+/dts-v1/;
+
+/include/ "AWM002.dtsi"
+
+/ {
+	compatible = "AWM002", "ralink,rt5350-soc";
+	model = "AsiaRF AWM002";
+
+	palmbus@10000000 {
+
+		spi@b00 {
+			m25p80@0 {
+				compatible = "mx25l3205d";
+				linux,modalias = "m25p80", "mx25l3205d";
+			};
+		};
+	};
+};
diff --git a/target/linux/ramips/dts/AWM002-8M.dtsi b/target/linux/ramips/dts/AWM002-8M.dtsi
new file mode 100644
index 0000000..d4cc7de
--- /dev/null
+++ b/target/linux/ramips/dts/AWM002-8M.dtsi
@@ -0,0 +1,18 @@
+/dts-v1/;
+
+/include/ "AWM002.dtsi"
+
+/ {
+	compatible = "AWM002", "ralink,rt5350-soc";
+	model = "AsiaRF AWM002";
+
+	palmbus@10000000 {
+
+		spi@b00 {
+			m25p80@0 {
+				compatible = "mx25l6405d";
+				linux,modalias = "m25p80", "mx25l6405d";
+			};
+		};
+	};
+};
diff --git a/target/linux/ramips/dts/AWM002-EVB-4M.dts b/target/linux/ramips/dts/AWM002-EVB-4M.dts
new file mode 100644
index 0000000..12f519d
--- /dev/null
+++ b/target/linux/ramips/dts/AWM002-EVB-4M.dts
@@ -0,0 +1,49 @@
+/dts-v1/;
+
+/include/ "AWM002-4M.dtsi"
+
+/ {
+	model = "AsiaRF AWM002 EVB";
+
+	gpio-leds {
+		compatible = "gpio-leds";
+		3g {
+			label = "awm002-evb:green:3g";
+			gpios = <&gpio0 9 1>;
+		};
+		gateway {
+			label = "awm002-evb:green:gateway";
+			gpios = <&gpio0 11 1>;
+		};
+		ap {
+			label = "awm002-evb:green:ap";
+			gpios = <&gpio0 12 1>;
+		};
+		wps {
+			label = "awm002-evb:green:wps";
+			gpios = <&gpio0 14 1>;
+		};
+		station {
+			label = "awm002-evb:green:station";
+			gpios = <&gpio0 13 1>;
+		};
+	};
+
+	gpio-keys-polled {
+		compatible = "gpio-keys-polled";
+		#address-cells = <1>;
+		#size-cells = <0>;
+		poll-interval = <20>;
+		reset_wps {
+			label = "reset_wps";
+			gpios = <&gpio0 10 1>;
+			linux,code = <0x198>;
+		};
+		mode {
+			label = "mode";
+			gpios = <&gpio0 7 1>;
+			linux,code = <0x32>;
+		};
+	};
+
+};
diff --git a/target/linux/ramips/dts/AWM002-EVB-8M.dts b/target/linux/ramips/dts/AWM002-EVB-8M.dts
new file mode 100644
index 0000000..0691f9f
--- /dev/null
+++ b/target/linux/ramips/dts/AWM002-EVB-8M.dts
@@ -0,0 +1,49 @@
+/dts-v1/;
+
+/include/ "AWM002-8M.dtsi"
+
+/ {
+	model = "AsiaRF AWM002 EVB";
+
+	gpio-leds {
+		compatible = "gpio-leds";
+		3g {
+			label = "awm002-evb:green:3g";
+			gpios = <&gpio0 9 1>;
+		};
+		gateway {
+			label = "awm002-evb:green:gateway";
+			gpios = <&gpio0 11 1>;
+		};
+		ap {
+			label = "awm002-evb:green:ap";
+			gpios = <&gpio0 12 1>;
+		};
+		wps {
+			label = "awm002-evb:green:wps";
+			gpios = <&gpio0 14 1>;
+		};
+		station {
+			label = "awm002-evb:green:station";
+			gpios = <&gpio0 13 1>;
+		};
+	};
+
+	gpio-keys-polled {
+		compatible = "gpio-keys-polled";
+		#address-cells = <1>;
+		#size-cells = <0>;
+		poll-interval = <20>;
+		reset_wps {
+			label = "reset_wps";
+			gpios = <&gpio0 10 1>;
+			linux,code = <0x198>;
+		};
+		mode {
+			label = "mode";
+			gpios = <&gpio0 7 1>;
+			linux,code = <0x32>;
+		};
+	};
+
+};
diff --git a/target/linux/ramips/dts/AWM002-8M.dts b/target/linux/ramips/dts/AWM002.dtsi
similarity index 94%
rename from target/linux/ramips/dts/AWM002-8M.dts
rename to target/linux/ramips/dts/AWM002.dtsi
index b112981..5fe2faa 100644
--- a/target/linux/ramips/dts/AWM002-8M.dts
+++ b/target/linux/ramips/dts/AWM002.dtsi
@@ -23,9 +23,7 @@
 			m25p80@0 {
 				#address-cells = <1>;
 				#size-cells = <1>;
-				compatible = "mx25l6405d";
 				reg = <0 0>;
-				linux,modalias = "m25p80", "mx25l6405d";
 				spi-max-frequency = <10000000>;
 
 				partition@0 {
